cpp/include/cudf/transform.hpp

```diff
@@ -83,5 +83,29 @@ std::pair<std::unique_ptr<rmm::device_buffer>, size_type> nans_to_nulls(
 std::pair<std::unique_ptr<rmm::device_buffer>, cudf::size_type> bools_to_mask(
   column_view const& input, rmm::mr::device_memory_resource* mr = rmm::mr::get_default_resource());
 
+/**
+ * @brief Encode the values of the given column as integers
+ *
+ * The encoded values are integers in the range [0, n), where `n`
+ * is the number of distinct values in the input column.
+ * The result column is such that keys[result[i]] == input[i],
+ * where `keys` is the set of distinct values in `input` in sorted order.
+ *
+ * Examples:
```

Comment:
Surround examples with `@code{.pseudo}` and `@endcode` doxygen tags.
https://github.com/rapidsai/cudf/blob/branch-0.15/cpp/docs/DOCUMENTATION.md#inline-examples

Reply:
Nice! Done